### PR TITLE
bazel 0.9.0 running `make bazel-build` updates

### PR DESCRIPTION
**Pull request by blakebarnett**
I'm not certain this is needed or correct, but makes my tree look good :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Go embed files?
+*.go-e
```
Comment (chrislovecnm) [inline .gitignore]: not a bad thing I assume, but what is this??

Comment (blakebarnett) [reply .gitignore]: I actually have no idea, but I had tons of them in my tree after doing the bazel build.

**Pull request merged**